crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -90,13 +89,10 @@ c_instance = C()
 
 reveal_type(c_instance.declared_and_bound)  # revealed: str | None
 
-# TODO: we currently plan to emit a diagnostic here. Note that both mypy
-# and pyright show no error in this case! So we may reconsider this in
-# the future, if it turns out to produce too many false positives.
```

Comment:
it would be useful to retain a comment here that mypy and pyright do not show an error in this case, but we deliberately deviate from them